File: src/main/java/no/nav/veilarbvedtaksstotte/client/dokument/SendDokumentDTO.java
package no.nav.veilarbvedtaksstotte.client.dokument;

import lombok.Data;
import lombok.experimental.Accessors;

import java.util.List;

@Data
@Accessors(chain = true)
public class SendDokumentDTO {
    String brukerFnr;
    MalType malType;
    String enhetId;
    String begrunnelse;
    List<String> opplysninger;

    public String getBrukerFnr() {
        return this.brukerFnr;
    }

    public MalType getMalType() {
        return this.malType;
    }

    public String getEnhetId() {
        return this.enhetId;
    }

    public String getBegrunnelse() {
        return this.begrunnelse;
    }

    public List<String> getOpplysninger() {
        return this.opplysninger;
    }
}

File: src/main/java/no/nav/veilarbvedtaksstotte/client/dokument/SendDokumentDTO.kt
package no.nav.veilarbvedtaksstotte.client.dokument

import no.nav.common.types.identer.EnhetId
import no.nav.common.types.identer.Fnr

data class SendDokumentDTO(
    val brukerFnr: Fnr,
    val malType: MalType,
    val enhetId: EnhetId,
    val begrunnelse: String?,
    val opplysninger: List<String>
)
